### DISCUSSION TITLE
PURGE

### DISCUSSION THREAD

**Discussion by comonier** [GriefPrevention/GriefPrevention#939] [Q&A] [answered]
Expiration:
      ChestClaimDays: 1
      UnusedClaimDays: 365
      AllClaims:
        DaysInactive: 365
        ExceptWhenOwnerHasTotalClaimBlocks: 10000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: true

First Question: that both lines ExceptWhenOwnerHasTotal/Bonus saves the Owner to NEVER have claim purged ?
Second Question: to Owner never have claim purged in that case need have both situations ? Total + Bonus? Or just one or other ?
Another Question: AutomaticNatureRestoration works likes World Edit commadn //regen ? or GriefPrevention dont need worldedit as dependence ? and do a own Regen  type.

**Comment by Jikoo** [comment, marked as answer]
1) Yes, if either criterion is met a claim owner's claims are permanent.
2) Just one or the other.
3) GP does its own "regeneration" that removes unnatural blocks. It won't be the same as world generation, and it won't do things like re-add ore veins the way a WorldEdit `//regen` would.
